composites/Plugin/DashboardWidget/components/WordpressFeed.js

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import colors from "../../../../style-guide/colors.json";
+
+/**
+ * @typedef  {Object}     Feed
+ * @property {string}     title       The title of the website.
+ * @property {string}     description A description of the website.
+ * @property {string}     link        A link to the website.
+ * @property {FeedItem[]} items     The items in the feed.
+ */
+
+/**
+ * @typedef  {Object} FeedItem
+ * @property {string} title       The title of the item.
+ * @property {string} content     The content of the item, will be HTML encoded.
+ * @property {string} description A summary of the content, will be HTML encoded.
+ * @property {string} link        A link to the item.
+ * @property {string} creator     The creator of the item.
+ * @property {string} date        The publication date of the item.
+ */
+
+const WordpressFeedContainer = styled.div`
+	box-sizing: border-box;
+	
+	p, a {
+		font-size: 14px;
+		margin: 0;
+	}
+`;
+
+const WordpressFeedHeader = styled.h3`
+	margin: 8px 0;
+	font-size: 1em;
+	padding: 0 8px;
+`;
+
+const WordpressFeedList = styled.ul`
+	margin: 0;
+	list-style: none;
+	padding: 0;
+`;
+
+const WordpressFeedLink = styled.a`
+`;
+
```

Comment:
not sure why we should use a styled component that doesn't set any style; why not use directly an `a` element in the JSX?